### PR TITLE
The distribution of the meta keywords

### PR DESCRIPTION
**Pull request by atomoc**
To remove duplicates - this is very important for the search engines. And to improve NodeBB.

### CODE REVIEW

**Comment by atomoc** [comment]
This is just the beginning. I plan to change the meta-description, add the content field and make a few more changes.

**Comment by atomoc** [comment]
Is there a reliable way to distinguish the home page from the internal pages?

File: src/meta/tags.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ module.exports = function(Meta) {
 					name: 'msapplication-square150x150logo',
 					content: Meta.config['brand:logo'] || ''
 				}];
+				if (link !== undefined) defaultTags.splice(4, 1);
```
Comment (atomoc) [inline src/meta/tags.js]: All is well?

File: src/views/admin/settings/general.tpl [hunk 2]
```diff
@@ -31,8 +31,8 @@
 			<label>Site Description</label>
 			<input type="text" class="form-control" placeholder="A short description about your community" data-field="description" /><br />
 
-			<label>Site Keywords</label>
-			<input type="text" class="form-control" placeholder="Keywords describing your community, comma-separated" data-field="keywords" /><br />
+			<label>Main Page Keywords</label>
```
Comment (atomoc) [inline src/views/admin/settings/general.tpl]: Or "Home Page"?

Comment (julianlam) [inline src/views/admin/settings/general.tpl]: "Keywords", is fine.

File: src/meta/tags.js (hunk 1 above)
Comment (julianlam) [inline src/meta/tags.js]: What's this for?

Comment (atomoc) [inline src/meta/tags.js]: To remove meta-keywords on all pages except the main.

Comment (julianlam) [inline src/meta/tags.js]: I don't believe leaving them on all pages would _harm_ SEO value... in either case, the correct course of action would be to add the `keywords` meta tag to `res.locals.metaTags` in `src/controllers/index.js`, `Controllers.home`. See how it is done in `src/controllers/topic.js`

Comment (atomoc) [inline src/meta/tags.js]: True, but something I did not get :)

**Comment by pitaj** [comment]
@julianlam @barisusakli @psychobunny You've got quiet a few very old PRs that should either be closed or updated. Please check this stuff out.

**Pull request closed without merging**